site/static/examples/pods-kueue/kueue-pod-group.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: sample-pod-
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+    kueue.x-k8s.io/pod-group-name: "sample-group"
+  annotations:
+    kueue.x-k8s.io/pod-group-total-count: "2"
+spec:
+  containers:
+  - name: sleep
+    image: busybox
+    command:
+      - sleep
+    args:
+      - 3s
+    resources:
+      requests:
+        cpu: 3
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: sample-pod-
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+    kueue.x-k8s.io/pod-group-name: "sample-group"
+  annotations:
+    kueue.x-k8s.io/pod-group-total-count: "2"
+spec:
+  containers:
+  - name: sleep
+    image: busybox
+    command:
+      - sleep
+    args:
+      - 3s
+    resources:
+      requests:
+        cpu: 3
```

Comment:
Should the definition of the pod group be there twice?

Reply:
It does not need to be, we use `generateName`, so we could just ask to run the kubectl create twice. 

However, I think from the docs point of view it can be beneficial to show they may differ, and play slightly different roles. I will suggest a change to modify one of them.

Reply:
Updated, PTAL